src/org/exist/storage/NativeValueIndex.java

```diff
@@ -1323,7 +1323,7 @@ private AtomicValue convertToAtomic( int xpathType, String value )
                 atomic = new StringValue( value ).convertTo( xpathType );
             }
             catch( final XPathException e ) {
-                LOG.warn( "Node value '" + value + "' cannot be converted to " + Type.getTypeName( xpathType ) );
+                LOG.debug( "Node value '" + value + "' cannot be converted to " + Type.getTypeName( xpathType ) );
```

Comment:
Thanks @wolfgangmm. I understand but I think in future we will need to develop a better error reporting mechanism to our users around indexing